src/isar/state_machine/state_machine.py

```diff
@@ -321,8 +286,39 @@ def _check_dependencies(self):
                 return False
         return True
 
+    def publish_telemetry_thread(self) -> None:
+        robot_id = settings.ROBOT_ID
+        PUBLISH_FREQUENCY = 1
```

Comment:
Should this be in the settings?

Reply:
The `PUBLISH_FREQUENCY` you mean? That makes sense. I can put it there.